### DISCUSSION TITLE
"Disk space is low" error despite ample free space - Unable to upload files desktop app

### DISCUSSION THREAD

**Discussion by syhxsthy36g** [nextcloud/all-in-one#5349] [Questions] [answered]
Hello,

I'm using TrueNAS Scale serveur:

- TrueNAS Scale version: TrueNAS-SCALE-24.04.2.2
- Nextcloud App version: 30.0.0
- Nextcloud Chart version: 2.0.17

I have 3TB of free space remaining (out of 4TB total) on my hard drive. However, when I try to synchronize files via the Nextcloud desktop app on Windows, I'm unable to transfer a file named MyVM.vdi (approximately 50GB in size). I get the error: "Disk space is low: Downloads that would reduce free space below 238 MB were skipped."

Strangely, I was able to successfully upload around 100 videos totaling 80GB. I still have plenty of space left on my user quota: "374GB/1TB". Additionally, my laptop, where the Nextcloud desktop app is installed, has approximately 30GB of free space.

Key points:

- Ample HDD storage: 3TB free on TrueNAS Scale (4TB total)
- User quota: 374GB used out of 1TB
- Successful smaller uploads: Recently uploaded 80GB of videos without issue
- Laptop free space: Approximately 30GB available
- Laptop OS : Windows 11 23H2
- Nextcloud desktop client : 3.14.1

Here's my Nextcloud configuration on TrueNAS:
![image](https://github.com/user-attachments/assets/16783657-6549-423f-9a86-7b3c9bbe3156)

Any help would be greatly appreciated. Please let me know if you need any additional information.

**Comment by szaimen** [comment, marked as answer]
Hi, as this seems to be unrelated to Nextcloud AIO, please open a thread for this at https://help.nextcloud.com. Thanks!

**Reply by syhxsthy36g** [reply]
Thank you for your quick response. My apologies, I mistakenly thought that support for Nextcloud on TrueNAS Scale was handled here. I'll post my question on the Nextcloud forums instead. Thanks again!
